src/platform/CMakeLists.txt

```diff
@@ -62,6 +62,10 @@ endif()
 add_library(msquic_platform STATIC ${SOURCES})
 
 if("${CX_PLATFORM}" STREQUAL "windows")
+    target_compile_definitions(msquic_platform PRIVATE
+        XDP_API_VERSION=3
+        XDP_INCLUDE_WINCOMMON
```

Comment:
How about we define these in the code, before the header include?

Reply:
I also doubt MsQuic needs to define `XDP_INCLUDE_WINCOMMON`, though you can if you want.

Reply:
Removing `XDP_INCLUDE_WINCOMMON` causes build error